source/guide_couchdb.rst

```diff
@@ -0,0 +1,148 @@
+=======
+
+All relevant legal information can be found here
+
+  * http://www.apache.org/licenses/LICENSE-2.0
+
+Prerequisites
+=============
+
+We're using :manual:`CouchDB <database-couchdb>` in the stable version 3:
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ uberspace tools version use couchdb 3
+  Selected couchdb version 3
+  The new configuration is adapted immediately. Minor updates will be applied automatically.
+  [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Installation
+============
+
+Uberspace provides the latest binaries, see :manual:`CouchDB <database-couchdb>` on how to interact with them. No extra installation required!
+
+Configuration
+=============
+
+Environment setup
+-----------------
+
+Next we need to create directories for our configuration and application data:
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ mkdir -p ~/etc/couchdb
+  [isabell@stardust ~]$ mkdir -p ~/opt/couchdb
+
+Application config
+------------------
+
+Create ``~/etc/couchdb/local.ini`` with the following content:
+
+.. warning:: Replace ``<username>`` with your username!
+
+.. code-block:: ini
+
+  [couchdb]
+  single_node=true
+  database_dir = /home/<username>/opt/couchdb/data
+  view_index_dir = /home/<username>/opt/couchdb/index
+
+  [chttpd]
+  port = 5984
+  bind_address = 0.0.0.0
+
+  [admins]
+  admin = SecureAdminPassword
+
+.. note:: Make sure to set your own super secure admin password!
+
+Setup daemon
+------------
+
+Create ``~/etc/services.d/couchdb.ini`` with the following content:
+
+.. warning:: Replace ``<username>`` with your username!
+
+.. code-block:: ini
+
+  [program:couchdb]
+  command=couchdb -couch_ini /opt/couchdb/etc/default.ini %(ENV_HOME)s/etc/couchdb/local.ini
+  autostart=yes
+  autorestart=yes
+
+.. include:: includes/web-domain-list.rst
+
+Finishing installation
+======================
+
+If everything looks fine, you should now be able to query CouchDB using ``localhost:5984``:
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ curl http://localhost:5984
+  {"couchdb":"Welcome","version":"3.1.1","git_sha":"CENSORED","uuid":"CENSORED","features":["access-ready","partitioned","pluggable-storage-engines","reshard","scheduler"],"vendor":{"name":"The Apache Software Foundation"}}
+
+Create database
+---------------
+
+Let's create a test database called ``testdb``:
+
+.. code-block:: bash
+
+  [isabell@stardust ~]$ curl -X PUT http://admin:SecureAdminPassword@localhost:5984/testdb
+  {"couchdb":"Welcome","version":"3.1.1","git_sha":"CENSORED","uuid":"CENSORED","features":["access-ready","partitioned","pluggable-storage-engines","reshard","scheduler"],"vendor":{"name":"The Apache Software Foundation"}}
+
+Best practices
+==============
+
+Security
+--------
+
+Change all default passwords. Especially the admin password within the config file ``~/etc/couchdb/local.ini``. Don't get hacked!
+
+Web Backend
+-----------
+
+To expose your CouchDB using a web backend (not recommended!):
```

Comment:
What do you think about moving the "not recommended!" into a `.. warning::` box. In there you could explain _why_ this is not recommended and when it might be necessary regardless.

Reply:
Moved into a warning box & added explanation on potential issues in commit 0c24f98415155a94192860de46cc98c20465f05e